webui/react/src/pages/FlatRuns/FlatRunActionButton.tsx

```diff
@@ -99,25 +103,23 @@ const FlatRunActionButton = ({
           break;
         }
         case 'ALL_EXCEPT': {
-          const filters = JSON.parse(tableFilterString);
+          const filterFormSet =
+            selection.type === 'ALL_EXCEPT'
+              ? (JSON.parse(tableFilterString) as FilterFormSet)
+              : INIT_FORMSET;
```

Comment:
`selection.type` is already confirmed to be `ALL_EXCEPT` in this branch of the codebase